mkdocs.yml

```diff
@@ -1,7 +1,30 @@
-site_name: PrairieLearn
-site_url: https://github.com/PrairieLearn/PrairieLearn
+site_name: PrairieLearn Docs
+site_url: https://prairielearn.readthedocs.io
 repo_url: https://github.com/PrairieLearn/PrairieLearn
-theme: readthedocs
+theme:
+  name: material
+  logo: assets/PL-logo.svg
+  favicon: assets/favicon.ico
+  icon:
+    repo: fontawesome/brands/github
+    edit: material/pencil
+  palette:
+    scheme: default
+    primary: custom
+  features:
+    - navigation.tabs
```

Comment:
TBD if I'm sold on this. I kinda prefer something closer to the current docs site:

```yaml
    - navigation.sections
    - navigation.expand
```